src/core/namespace.c

```diff
@@ -56,10 +56,10 @@ typedef enum MountMode {
         PRIVATE_TMP,
         PRIVATE_VAR_TMP,
         PRIVATE_DEV,
+        READONLY,
         BIND_DEV,
         SYSFS,
         PROCFS,
-        READONLY,
```

Comment:
Hmm, I see I had a previous version of this patch without the propagate read-only flag to previous mounts. Yes this one is unnecessary.